types/externs.d.ts

```diff
@@ -175,6 +175,12 @@ declare global {
       throttlingMethod?: 'devtools'|'simulate'|'provided';
       /** The throttling config settings. */
       throttling?: ThrottlingSettings;
+      /** A user-facing string to display in Runtime Settings for the Device row */
```

Comment:
sorry to bikeshed, but we are going to be stuck with these indefinitely, so... 😬 

Exploring the possibility space:
- should these be nested under an object of provided override strings? Keeps them all in one place, allows for a home for future override strings
- didn't we eventually decide "provided" is confusing, since it's not always clear who is providing what? What about something like `override*`? `display*`?